.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,16 +2,8 @@
 
 WIP: work in progress
 
-Consider prefix your PR message with an emoticon
-  🐛 bugfix
-  ✨ new feature
-  🔨 refactoring
-  🏗️ maintenance
-  📚 documentation
-
-or get your emoji from https://gitmoji.dev/
-
-and append (⚠️ devops) if changes in devops configuration required before deploying
+Consider prefix your PR message with an emoji from https://gitmoji.dev/
```

Comment:
actually would keep the examples in there. it's faster to copy/paste them than going to the website...